File: RECORRIDO EN MATRICES/RECORRIDOMATRICES.cpp
#include <iostream>
using namespace std;

int main() {
    int matriz[3][3] = {
        {1, 2, 3},
        {4, 5, 6},
        {7, 8, 9}
    };

    // Recorrido por FILAS
    cout << "Recorrido por FILAS:\n";
    for (int i = 0; i < 3; i++) {
        for (int j = 0; j < 3; j++) {
            cout << matriz[i][j] << " ";
        }
        cout << endl;
    }

    // Recorrido por COLUMNAS
    cout << "\nRecorrido por COLUMNAS:\n";
    for (int j = 0; j < 3; j++) {
        for (int i = 0; i < 3; i++) {
            cout << matriz[i][j] << " ";
        }
        cout << endl;
    }

    return 0;
}

File: RECORRIDO EN MATRICES/RECORRIDOMATRICES.py

matriz = [
    [1, 2, 3],   
    [4, 5, 6],   
    [7, 8, 9]    
]

print("Recorrido por FILAS:")
for i in range(len(matriz)):          
    for j in range(len(matriz[i])):   
        print(matriz[i][j], end=" ")
    print() 


print("\nRecorrido por COLUMNAS:")
for j in range(len(matriz[0])):       
    for i in range(len(matriz)):      
        print(matriz[i][j], end=" ")
    print()  
